app/1.0/start/toolbox/add-elements.md

```diff
@@ -8,79 +8,67 @@ subtitle: "Build an app with App Toolbox"
 Now that you've added a new view to your application, you can start building
 out the details of that view.
 
-In the process, you'll likely want to turn
-to some off-the-shelf components, for example from the
-[Polymer Element Catalog][catalog] or community catalogs like
-[customelements.io][ceio].
+One of the appeals of the web component approach to web development 
+is the ecosystem. Rather than building everything in your app from scratch,
+you can find and use off-the-shelf elements that suit your needs. Two examples
+of places you can find reusable components are the 
+[Polymer Element Catalog][catalog] (elements built by the Polymer team) and 
+[customelements.io][ceio] (elements built by the web components community). 
 
-## Ensure bower is installed
+In this tutorial you install a third-party component and use it in the new
```

Comment:
Here (and in next header and paragraph) "third-party' should be "off-the-shelf" or something like that. Third party implies someone other than us, the writers, and you, the user. And these components are supplied by us, the writers. Well, virtually speaking.